composer.json

```diff
@@ -51,7 +61,7 @@
         "friendsofsymfony/jsrouting-bundle": "^2.3",
         "friendsofsymfony/rest-bundle": "^2.8 || ^3.0",
         "gedmo/doctrine-extensions": "^2.4 || ^3.0",
-        "goodby/csv": "^1.3",
+        "goodby/csv": "dev-patch-2 as 1.4.0",
```

Comment:
As I think goodby csv is unmaintained and we will not get a release for this fix we have to create a decission here. The package is only used at this place in our code: https://github.com/sulu/sulu/blob/2.x/src/Sulu/Component/Rest/Csv/CsvHandler.php

A. Replace CSV Handler with a raw fputcsv and so implement that logic ourself.
B. Fork the goodby/csv and create a compatible version.

/cc @wachterjohannes what would you do?

Reply:
@alexander-schranz i would try to replace it with standard csv functions of php - `fputcsv` for example - the streaming functionality of `goodby/csv` cant be really used by the the handler because we load all the data in front. so throw it away 😂

Reply:
As discussed we fork it to https://github.com/handcraftedinthealps/goodby-csv.